CODEOWNERS

```diff
@@ -1,5 +1,9 @@
 # Prevent accidentially touching CSS subtree
-/css/tools/apiclient/ @plinss @jgraham
+/css/tools/apiclient/ @plinss @web-platform-tests/wpt-core-team
+
+# Require review for changes that often need an RFC
+/resources/testdriver* @web-platform-tests/wpt-core-team
+/resources/testharness* @web-platform-tests/wpt-core-team
```

Comment:
I guess my one question here is whether we want to make this all of `/resources`, and go back to a stronger separation with `/common`?

Reply:
I took a quick look and came up with "no" as the answer since I don't think idlharness.js is important enough to require core team review.

But if someone does the work of moving more stuff to `/common` I'd be fine with treating all of `/resources` as special.